tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/OrderItemDeleteSteps.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Data.SqlClient;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Requests;
+using NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Responses;
+using NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Steps.Common;
+using NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Support;
```

Comment:
Do we need all of these `using` statements?

Reply:
removed